bin/load-queued-items.ts

```diff
@@ -0,0 +1,149 @@
+#! /usr/bin/env ts-node
+
+/**
+ * Usage: npm run load <feed_id>
+ */
+
+import knex from 'knex';
+
+const connection = {
+  host: '<host>',
+  user: '<user>',
+  password: '<password>',
+  port: 3306,
+  database: 'readitla_ril-tmp',
+  charset: 'utf8mb4',
+};
+
+const dbClient = knex({
+  client: 'mysql',
+  connection,
+  pool: {
+    /**
+     * Explicitly set the session timezone. We don't want to take any chances with this
+     */
+    afterCreate: (connection, callback) => {
+      connection.query(`SET time_zone = 'US/Central';`, (err) => {
+        callback(err, connection);
+      });
+    },
+  },
+});
+
+const GET_NEXT_QUEUED_ITEM =
+  'SELECT\n' +
+  '                  q.queued_id,\n' +
+  '                  q.relevance_length,\n' +
+  '                  q.weight,\n' +
+  '                  q.feed_id,\n' +
+  '                  q.prospect_id,\n' +
+  '                  q.resolved_id,\n' +
+  '                  CASE\n' +
+  "                    WHEN q.relevance_length = 'day' THEN :dayWeight\n" +
+  "                    WHEN q.relevance_length = 'week' THEN :weekWeight\n" +
+  "                    WHEN q.relevance_length = 'forever' THEN :foreverWeight\n" +
+  '                  END AS time_weight,\n' +
+  '                  q.time_added\n' +
+  '                FROM `readitla_ril-tmp`.curated_feed_queued_items q\n' +
+  '                WHERE q.feed_id = :feedId\n' +
+  "                AND q.status = 'ready'\n" +
+  '                ORDER BY q.weight DESC, time_weight DESC, q.time_added ASC\n' +
+  '                LIMIT 1;';
+
+async function getNextQueuedItem(feedId) {
+  const dayWeight = getRandomInt(1, 101);
```

Comment:
thinking why order matters here as the day and week weights are all assigned randomly? 

to confirm: we just want assign all items in the queue and we are not prioritizing them based on if its `relevant length` is `day` or `week`

Reply:
replicating the logic in https://github.com/Pocket/Web/blob/main/processes/curation/add_feed_items.py